metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -213,6 +221,13 @@ func NewContainerMetadataEnricher(
 					}
 				}
 
+				if s, ok := statuses[container.Name]; ok {
+					split := strings.Index(s.ContainerID, "://")
+					if split != -1 {
+						meta.Put("container.id", s.ContainerID[split+3:])
```

Comment:
could you please add a comment with ab example of `statuses[container.Name]` value, that could explain why those offsets are used

Reply:
I added a comment in https://github.com/elastic/beats/pull/29560/commits/755b1f23cd0eb218964b736e3250499f87fc55cc